client/state/sites/selectors.js

```diff
@@ -123,3 +123,46 @@ export function getSiteByUrl( state, url ) {
 
 	return site;
 }
+
+/**
+ * Returns a site's plan object by site ID.
+ *
+ * The difference between this selector and sites/plans/getPlansBySite is that the latter selectors works
+ * with the /sites/$site/plans endpoint while the former selectors works with /sites/$site endpoint.
+ * Query these endpoints to see if you need the first or the second one.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {?Object}        Site's plan object
+ */
+export function getSitePlan( state, siteId ) {
+	const site = getSite( state, siteId );
+
+	if ( ! site ) {
+		return null;
+	}
+
+	return site.plan;
+}
+
+/**
+ * Returns true if site is currently subscribed to supplied plan and false otherwise.
+ *
+ * @param  {Object}   state         Global state tree
+ * @param  {Number}   siteId        Site ID
+ * @param  {Number}   planProductId Plan product_id
```

Comment:
It is snake_case in API result:
![selection_019](https://cloud.githubusercontent.com/assets/4988512/16130562/1ccfd9b4-340a-11e6-8549-4b0abafa0b5f.jpg)